grpc/service_config/service_config.proto

```diff
@@ -607,8 +624,12 @@ message LoadBalancingConfig {
         [json_name = "xds_cluster_resolver_experimental"];
     XdsClusterImplLoadBalancingPolicyConfig xds_cluster_impl_experimental = 12
         [json_name = "xds_cluster_impl_experimental"];
+    XdsWrrLocalityLoadBalancingPolicyConfig xds_wrr_locality_experimental = 16
+        [json_name = "xds_wrr_locality_experimental"];
```

Comment:
I don't really mind, but why intentionally order this way vs. by tag #?

Reply:
It was incorrect to put the `xds_wrr_locality` and `least_request` policies under the "Deprecated xDS-related policies" comment on line 634.  These are not deprecated.

It also makes sense to put the `outlier_detection` under the "REMAINING POLICIES ARE EXPERIMENTAL" comment on line 608.

Reply:
Oh I mean _within this section_, why not order by tag #?  This one entry (16) was added between 12 and 13, and not next to 17, all in this section.

Reply:
Ah, okay -- sorry, I misunderstood your question.

I'm ordering all of the xDS policies here by their position in the LB policy tree, from top to bottom.

Reply:
Seems reasonable to me.